addons/l10n_ae/data/l10n_ae_chart_data.xml

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <menuitem id="account_reports_ae_statements_menu" name="UAE" parent="account.menu_finance_reports" sequence="0" groups="account.group_account_user"/>
```

Comment:
? Nothing put anyways in community?

Reply:
@jco-odoo almost all modules follows this convention so i had follow this.

Like this,
community:
https://github.com/odoo/odoo/blob/8f4b98e67fe49516c0c830524de351db67c58357/addons/l10n_be/data/menuitem_data.xml#L3

enterprise:
https://github.com/odoo/enterprise/blob/0cd24d973cf96ccb70c5ea280ff95ea940c831e6/l10n_be_reports/data/account_financial_html_report_data.xml#L11